openlane/scripts/odbpy/io_place.py

```diff
@@ -56,7 +56,12 @@ def equally_spaced_sequence(side, side_pin_placement, possible_locations):
 
     if total_pin_count > tracks:
         print(
-            f"There are more pins/virtual_pins: {total_pin_count}, than places to put them: {tracks} in the {side} side. Try making your floorplan area larger."
+            f"[ERROR] The {side} side of the floorplan doesn't have enough slots for all the pins: {total_pin_count}/{tracks}.",
```

Comment:
Would prefer more verbosity when saying `total_pin_count/tracks`

Reply:
Will include with the next set of changes